test/functional/test-custom-element.js

```diff
@@ -230,6 +242,24 @@ describe('CustomElement', () => {
     expect(testElementBuildCallback.callCount).to.equal(1);
   });
 
+  it('Element - build NOT allowed when in template', () => {
+    const element = new ElementClass();
+    expect(element.classList.contains('-amp-element')).to.equal(true);
+    expect(element.isBuilt()).to.equal(false);
+    expect(element.classList.contains('-amp-notbuilt')).to.equal(true);
+    expect(element.classList.contains('amp-notbuilt')).to.equal(true);
```

Comment:
use `expect(element).to.hav.class('-amp-notbuilt')` instead (here and any other class existence check)